llvm-sanitizers.md

```diff
@@ -0,0 +1,24 @@
+# LLVM TSAN / ASAN
+
+For multithreaded and low-level unsafe interfacing server code, the ability to use LLVM:s [ThreadSanitizer](https://clang.llvm.org/docs/ThreadSanitizer.html) and  
+[AddressSanitizer](https://clang.llvm.org/docs/AddressSanitizer.html) can help troubleshoot invalid thread usage and invalid usage/access of memory.
+
+There is a [blog post](https://swift.org/blog/tsan-support-on-linux/) outlining the usage of TSAN.
+
+The short story is to use the swiftc command line options `-sanitize=address` and `-santize=thread` to each respective tool.
+
+Also for Swift Package Manager projects you can use `--sanitize` at the command line, e.g.:
+
+    swift build --sanitize=address
```

Comment:
make into code snippet?

```bash
swift build --sanitize=address
```

Reply:
Four spaces make code too. See here:

    This is just indented with for spaces

whereas

```
that is done using treble backticks
```

.

Reply:
Yeah, it works though I'm not sure about adopting it throughout... it's a bit easier to format code consistently with the ``` fences (and review if indented correctly etc)